about.md

```diff
@@ -3,9 +3,9 @@ title: "About"
 description: "Learn about Coder, our mission, and why to choose us."
 ---
 
-Coder moves software development to the cloud, centralizing your organizations'
-development initiatives and offering improvements in development velocity and
-enterprise security.
+Coder is a self-hosted platform for DevOps, platform, and engineering teams.
+With Coder, organizations can securely provision remote IDEs and related resources
+on their infrastructure. We call this logical group of resources a developer workspace.
 
```

Comment:
@bpmct wdyt instead

> Coder is a self-hosted platform that allows organizations to securely provision developer workspaces (featuring remote IDEs and all related resources) for devOps, platform, and software engineering teams.

Reply:
I like it. However, I think it should be `DevOps`